Updated go code templates to add meta data and tags.

mboshu opened the pull request.

mboshu committed:
Updated go code templates to add meta data and tags.

david-schrenker commented on examples/basic/BasicComplex/BasicSendComplex.go:
```diff
@@ -55,6 +55,12 @@ func main() {
+	basic.Metadatas = append(basic.Metadatas, message.NewMetadata("MyMetaDatra", "I am meta data"))
+
+	//Add message tags
+	basic.Tags = append(basic.Tags, "I am a Tag")
```
add tag "go-Example"

david-schrenker commented on examples/bulk/BulkComplex/BulkComplex.go:
```diff
@@ -45,6 +45,11 @@ func main() {
+	bulk.AddMetadata("testMetadata", "I am meta data")
+
+	//Add message tags
+	bulk.Tags = append(bulk.Tags, "I am a Tag")
```
add tag "go-Example"

david-schrenker commented on examples/basic/BasicWithMetadata/BasicWithMetadata.go:
```diff
@@ -0,0 +1,33 @@
+package main
```
please remove

david-schrenker commented on examples/basic/BasicWithTags/BasicWithTags.go:
```diff
@@ -0,0 +1,33 @@
+package main
```
please remove

david-schrenker commented on README.md:
```diff
@@ -144,6 +144,12 @@ This example demonstrates how to add a file attachment to your message.
 ### [Basic send with custom email headers](https://github.com/socketlabs/socketlabs-go/blob/master/examples/basic/BasicWithCustomHeaders/BasicWithCustomHeaders.go)
 This example demonstrates how to add custom headers to your email message.
 
+### [Basic send with meta data](https://github.com/socketlabs/socketlabs-go/blob/master/examples/basic/BasicWithMetadata/BasicWithMetadata.go)
```
please remove

david-schrenker commented on injectionapi/core/serialization/MetadataJson.go:
```diff
@@ -0,0 +1,12 @@
+type MetadataJson struct {
+
+	// Gets or sets the meta data name
+	Name string `json:"Name"`
```
change to "key"

david-schrenker commented on injectionapi/message/metadata.go:
```diff
@@ -0,0 +1,29 @@
+
+// CustomHeader  info
+type Metadata struct {
+	Name  string
```
change to "key"

david-schrenker commented on .vs/VSWorkspaceState.json:
```diff
@@ -0,0 +1,16 @@
+{
```
add '.vs/*' to .gitignore

david-schrenker committed:
removing .vs folder

david-schrenker committed:
updating version

mboshu committed:
Removed unneeded tests
Changed 'name' to 'key' for metadata
Added go example tags
Changed 'Metadatas' to 'Metadata'
Added items to gitignore.

mboshu committed:
Merged changes
